apps/services/handlers.py

```diff
@@ -1,279 +1,183 @@
-                     summary="Get user request list",
-                     responses={200: {"description": "Successful get university user request list response"}},
-                     tags=["Student dashboard"]
-                     )
-async def read_user_request_list(university_id: int, user=Depends(get_current_user)):
-    query = select(user_request_list_view).where(user_request_list_view.c.user_id == user.user_id,
-                                                 user_request_list_view.c.university_id == university_id)
-    return {
-        "data": await database.fetch_all(query),
-        "message": "Got user requests list"
-    }
-
-
-@services_router.post("/{university_id}/user-request/",
-                      name="create_user_request",
-                      response_model=JSENDOutSchema[CreateUserRequestOut],
-                      summary="Create user request",
-                      responses={200: {"description": "Successful create user request response"}},
-                      tags=["Student dashboard"])
-async def create_user_request(university_id: int, user_request: CreateUserRequestIn, user=Depends(get_current_user)):
-    """
-    **Create user request**
 
-    **Path**:
-    - **university_id**: user university id
-
-    **Input**:
-    - **service_id**: service id in database, required
-    - **comment*: comment for the creating user request
-
-    **Return**: user request id; request status id
-    """
-    query = select(UserFaculty).where(UserFaculty.user_id == user.user_id)
-    user_faculty_result = await database.fetch_one(query)
-    query = insert(UserRequest).values(date_created=datetime.now(),
-                                       comment=user_request.comment,
-                                       user_id=user.user_id,
-                                       service_id=user_request.service_id,
-                                       faculty_id=user_faculty_result.faculty_id,
-                                       university_id=university_id,
-                                       status_id=STATUS_MAPPING.get("Розглядається"))
-
-    last_record_id = await database.execute(query)
-
-    query = select(user_request_booking_hostel_view).where(user_request_booking_hostel_view.c.user_id == user.user_id,
-                                                           user_request_booking_hostel_view.c.university_id == university_id)
-
-    result = await database.fetch_one(query)
-
-    prepared_data = {
-        "context": result,
-        "service_id": user_request.service_id,
-        "user_request_id": last_record_id
-    }
-
-    await create_user_document(**prepared_data)
-
-    return {
-        "data": {
+    async def read_user_request_list(
+            self,
+            *,
+            request: Request,
+            university_id: int,
+            user: UserOut,
+            session: AsyncSession
+    ):
+        return await request_existence_list_service.list(
```

Comment:
don't forget to rename request_existence_list_service

Reply:
ok